coalib/parsing/DefaultArgParser.py

```diff
@@ -48,13 +48,30 @@ def default_arg_parser(formatter_class=None):
     else:
         parser_type = 'coala'
 
+    description = """
+coala provides a common command-line interface for linting and fixing all your
+code, regardless of the programming languages you use.
+
+To find out what kind of analysis offers for the languages you use, visit
+<https://github.com/coala-analyzer/bear-docs/blob/master/README.rst#supported-languages>
+or run:
+
+    $ coala --show-bears --filter-by-language C Python
+
+To perform code analysis, simply specify the analysis routines (bears) and the
+files you want it to run on:
```

Comment:
Mention that the following is an example? `to run on. For example:` (wouldn't take any extra lines)